src/AlgebraicGeometry/Schemes/AffineSchemes/Morphisms/Methods.jl

```diff
@@ -208,22 +208,12 @@ function Base.show(io::IO, ::MIME"text/plain", f::AbsSpecMor)
 end
 
 function Base.show(io::IO, f::AbsSpecMor)
-  io = pretty(io)
   if get(io, :supercompact, false)
     print(io, "Affine scheme morphism")
   else
-    X = domain(f)
-    Y = codomain(f)
+    io = pretty(io)
     print(io, "Hom: ")
-    if typeof(X) <: Union{PrincipalOpenSubset, AffineVariety{ <:Field, <: MPolyAnyRing}, <:Spec{<:Field, <:MPolyAnyRing}}  # Take care of the case where the domain is not given as a V(bla)
-      print(io, Lowercase())
```

Comment:
This case was checking for specific types of `X` to decide whether or not to issue `Lowercase()` -- that's a bad code smell, don't do that! Here, the proper fix is to ensure that the sub-object issues `LowercaseOff` if necessary (I've done on such change elsewhere in this PR).

@StevellM There is more printing code for Schemes which performs similar checks that should be changed. I'll write to you about this separately on Slack.